### PR TITLE
Moe Jangda - Task Manager Project

### PR DESCRIPTION
**Pull request by mistermoe**

### CODE REVIEW

**Comment by gilbert** [comment]
Your peer review partner is Jose: https://github.com/makersquare/ruby-course/pull/108

File: client.rb [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+    elsif(@@user_command == 'list')
+      puts
+      puts
+      puts " PID      Project Name".colorize(:mode => :underline)
+      @@project_list.projects.each {|project| puts "  #{project.id}       #{project.project_name}"}
+      # @@project_list.list_projects
+      puts
+
+  elsif (@@user_command.include? 'add')
```
Comment (Chakaitos) [inline client.rb:53]: I would suggest creating methods for all commands just like the 'help' method for easier comprehension of the code and more abstraction and calling them after the elsif for example:

elsif (@@user_command.include? 'add')
-      PM.add(*params)

---

def self.add(description, priority, pid)
 #add 2 take out the trash 1
-     puts "#{@@user_command}".colorize(:yellow)
-     pid = @@user_command.slice(4).to_i
-     priority = @@user_command.slice(@@user_command.length - 1).to_i
-     description = @@user_command.slice(6..@@user_command.length - 2)
-     @@project_list.add_task_to_project(description, priority, pid)
- ```
  chosen_project =  @@project_list.projects.find{|project| project.id == pid}
  ```
-    puts
-    puts "task was added to the #{chosen_project.project_name} project".colorize(:yellow)
-    puts

end

File: client.rb [hunk 2]
```diff
@@ -0,0 +1,98 @@
+require_relative 'lib/task-manager.rb'
+require 'colorize'
```
Comment (Chakaitos) [inline client.rb:2]: Great job... I like the use of the 'colorize' gem

File: client.rb (hunk 1 above)
Comment (mistermoe) [inline client.rb:53]: good point!

**Pull request closed without merging**